docs/src/install.md

```diff
@@ -1,57 +1,72 @@
-# Getting started: Installation and testing your install
+# Getting started
 
-Most users probably want to start with the Images package, which bundles
+Most users probably want to start with the `Images.jl` package, which bundles
 much (but not all) of the functionality in JuliaImages.
 
 ## Installation
 
-Install Images via the [package manager](https://docs.julialang.org/en/v1/stdlib/Pkg/),
+Images (and possibly some additional packages) may be all you need to manipulate images programmatically.
+You can install `Images.jl` via the [package manager](https://docs.julialang.org/en/v1/stdlib/Pkg/),
 
 ```julia
 (v1.0) pkg> add Images
 ```
 
-This will also install many dependencies.
-
-Images (and possibly some additional packages) may be all you need to manipulate images programmatically.
-However, most users will want to take one or two additional steps:
-ensuring that you can load and display images.
-
 !!! note
     People in some regions such as China might fail to install/precompile `Images` due to poor network
     status. Using proxy/VPN that has stable connection to Amazon S3 and Github can solve this issue.
 
 ## Loading your first image
 
-When testing ideas or just following along with the documentation, it can be
-useful to have some images to work with.
-The [TestImages](https://github.com/JuliaImages/TestImages.jl) package bundles several "standard" images for you.
-If you haven't already installed it, use `pkg> add TestImages`.
+If this is your first time working with images in julia, it's likely that you'll need to install some
```

Comment:
FileIO no longer prompts to give installation suggestions, so I add this part at the beginning of this section.